src/pages/HomePage/HomePage.js

```diff
@@ -5,7 +5,9 @@ import Carousel from '../../components/Carousel/Carousel'
 import brandImage1 from '../../assets/Images/loreal_logo.jpg'
 import brandImage2 from '../../assets/Images/lakme_logo.jpg'
 import brandImage4 from '../../assets/Images/maybelline_logo.png'
-import brandImage5 from '../../assets/Images/Wishper_logo.png'
+import brandImage5 from '../../assets/Images/mamaearth-logo.jpg'
+import brandImage6 from '../../assets/Images/wow-logo.png'
+import brandImage7 from '../../assets/Images/Wishper_logo.png'
 import offerImage1 from '../../assets/Images/offerImage1.jpg'
 import offerImage2 from '../../assets/Images/offerImage2.jpg'
```

Comment:
you can keep all imports in one file and take it from there. readability wont get hampered by doing that